### PR TITLE
Locales

### PR DESCRIPTION
**Pull request by webpro**
Hi,

Here are additions to existing locales, and a new locale for Italian ("it").

Note: I've also added suggested modifications for existing locales for you to consider. You can cherry pick some improvements, or ignore them completely if you want to.

Let me know if this works for you.

### CODE REVIEW

**Comment by dmitry-mukhin** [comment]
Lars, this looks like automated translation. We cannot accept that.

**Comment by webpro** [comment]
Hi @dmitry-mukhin. Translations are not automated. The texts have been translated by a professional company. Please let me know which locale or which part you think is automated.

**Comment by dmitry-mukhin** [comment]
Well, I just looked at Russian (my native), and it doesn't look like the translation was done by someone who knows the language or problem domain.

for example "download" is used where "upload" should be.

**Comment by webpro** [comment]
OK that's an issue. Not sure what went wrong. Please keep in mind that there are three separate commits. The changes for the RU locale, which you probably want to ignore, are in the third commit. The first two commits are far more interesting.

Any chance you can get someone familiar with the other languages to look at them? I truly think they're great additions to your locales.

File: app/assets/javascripts/uploadcare/locale/de.js.coffee [hunk 1]
```diff
@@ -1,96 +1,119 @@
...
+          title: 'Dateien aus dem Netz'
+          line1: 'Schnappen Sie sich irgendeine Datei aus dem Netz.'
           line2: 'Geben Sie einfach den Link an.'
-          input: 'Bitte geben Sie den Link hier an...'
+          input: 'Fügen Sie Ihren Link hier ein...'
           button: 'Hochladen'
+        camera:
+          capture: 'Machen Sie ein Foto'
+          mirror: 'Spiegel'
+          retry: 'Genehmigungen erneut anfordern'
```
Comment (dmitry-mukhin) [inline app/assets/javascripts/uploadcare/locale/de.js.coffee]: Genehmigungen -> Berechtigungen

**Comment by dmitry-mukhin** [comment]
@webpro for the moment I would ask you to revert 678dc4f, and change two words in 'de' and 'ru'

Also, if it's not too much to ask, remove unnecessary strings where translation match English (VK, Evernote, Uploadcare.com, etc.)

**Comment by webpro** [comment]
OK done. I haven't removed the English-matching translations (why are they an issue?).

**Comment by homm** [comment]
@webpro because they are included in result js library and adds size without any benefits.

**Comment by webpro** [comment]
OK, done (please also look into https://github.com/uploadcare/social-l10n/pull/4)

**Pull request merged**